cmd/demo/node.go

```diff
@@ -488,3 +514,16 @@ func Setup() {
 	}
 	backend = b
 }
+
+//etherToWei converts amount in "ether" (represented as decimal) to "wei" (represented as integer)
+//It can provide exact results for values in the range of 1e-18 to 1e9
```

Comment:
Please leave one space after the `//` and terminate all sentences with a `.`.

Reply:
Addressed in this commit - e10c196.